TeraProxyGUIWithConsole.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+./node_modules/electron/dist/electron --high-dpi-support=1 --force-device-scale-factor=1 --js-flags="--use-strict" ./bin/index.js
+
+echo "Press return to continue..."
+read
```

Comment:
See my comment above. These instructions are useless. Just `exec` the electron command instead.